lib/web_ui/lib/initialization.dart

```diff
@@ -112,10 +112,24 @@ set debugEmulateFlutterTesterEnvironment(bool value) {
     engine.window.webOnlyDebugPhysicalSizeOverride =
         logicalSize * window.devicePixelRatio;
   }
+  debugDisableFontFallbacks = value;
 }
 
 bool _debugEmulateFlutterTesterEnvironment = false;
 
+/// Whether to disable the font fallback system.
+///
+/// We need to disable font fallbacks for some framework tests because
+/// Flutter error messages may contain an arrow symbol which is not
+/// covered by ASCII fonts. This causes us to try to download the
+/// Noto Sans Symbols font, which kicks off a `Timer` which doesn't
+/// complete before the Widget tree is disposed (this is by design).
+bool get debugDisableFontFallbacks => _debugDisableFontFallbacks;
```

Comment:
Can this be moved into `engine.dart`? Otherwise, it will be publicly visible to Flutter developers from `dart:ui` imports.

Reply:
Done.